src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,27 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book1 = new Book("Book #1", BigDecimal.valueOf(10));
```

Comment:
why book1?

Reply:
> why book1?

Oh, at first I added several books, and then I decided to leave only one and forgot to rename it, I'll have to do that, thanks